ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CoveragesPageHeader.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { PageSection, Text, Title } from '@patternfly/react-core';
+
+import PageTitle from 'Components/PageTitle';
+
+function CoveragesPageHeader() {
+    return (
+        <>
+            <PageTitle title="Compliance coverage" />
```

Comment:
When you need a study break, can you add to your doc a proposal for page titles for browser history?
For example,
* Compliance coverage - Profile
* Compliance coverage - Checks
* Compliance coverage - Check
* Compliance coverage - Clusters
* Compliance coverage - Cluster

Access Control is an example similar to the above.

A question which you can either defer or explore is whether Clusters and Cluster are ambiguous if Benchmark is added in the future.

Reply:
I guess "Page titles" would also be a good addition to the [Frontend Design Doc checklist](https://docs.google.com/document/d/1g2VnZ_gJLeKAQSWyNiHF_ovf5JwW76B7vbVfaERyoGU/edit)